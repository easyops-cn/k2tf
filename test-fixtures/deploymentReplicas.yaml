apiVersion: apps/v1
kind: Deployment
metadata:
  labels:
    app: backend-api
  name: backend-api
  namespace: default
spec:
  replicas: 0
  selector:
    matchLabels:
      app: backend-api
  template:
    metadata:
      labels:
        app: backend-api
    spec:
      containers:
        - name: api
          ports:
            - containerPort: 443
              protocol: TCP
          image: gcr.io/project/backend-api:0.3.15
