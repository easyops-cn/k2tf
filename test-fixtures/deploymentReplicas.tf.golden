resource "kubernetes_deployment" "backend_api" {
  metadata {
    name      = "backend-api"
    namespace = "default"
    labels = {
      app = "backend-api"
    }
  }
  spec {
    replicas = 0
    selector {
      match_labels = {
        app = "backend-api"
      }
    }
    template {
      metadata {
        labels = {
          app = "backend-api"
        }
      }
      spec {
        container {
          name    = "api"
          image   = "gcr.io/project/backend-api:0.3.15"
          port {
            container_port = 443
            protocol       = "TCP"
          }
        }
      }
    }
  }
}
